lab_4_retrieval_w_clustering/main.py

```diff
@@ -4,8 +4,16 @@
 Vector search with clusterization
 """
 
+from lab_2_retrieval_w_bm25.main import calculate_bm25
+
```

Comment:
ignores before imports

Reply:
What do you mean?

Reply:
pylint ignores (line 10) should be before imports (lines 6-8)